types/function-bind/ts3.2/implementation.d.ts

```diff
@@ -1,183 +0,0 @@
-//#region bind():
-/**
- * For a given function, creates a bound function that has the same body as the original function.
- * The this object of the bound function is associated with the specified object, and has the specified initial parameters.
- * @param thisArg The object to be used as the this object.
- * @param args Arguments to bind to the parameters of the function.
- */
-declare function bind<T, A extends any[], R>(this: (this: T, ...args: A) => R, thisArg: T): (...args: A) => R;
```

Comment:
**TS 3.3+** supports `ThisParameterType` and `OmitThisParameter`:
<https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b21787307a30f1d7b7d8e80bc58517e3576266bf/types/function-bind/ts3.9/implementation.d.ts#L8>